src/main/java/com/devonfw/quarkus/productmanagement/domain/model/ProductEntity.java

```diff
@@ -1,30 +1,29 @@
-package com.devonfw.demoquarkus.domain.model;
+package com.devonfw.quarkus.productmanagement.domain.model;
+
+import java.math.BigDecimal;
 
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
 import javax.persistence.Table;
 
+import com.devonfw.quarkus.general.domain.model.ApplicationPersistenceEntity;
+
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 @Entity
-@Table(name="Animal")
+@Table(name = "Product")
 // A JPA entity requires at least 2 things @Entity annotation and an ID
 // by default, the DB table will have the same name as our class
-public class AnimalEntity {
-
-  @Id
-  @GeneratedValue
-  private Long id;
+public class ProductEntity extends ApplicationPersistenceEntity {
 
-  private String name;
+  private String title;
 
-  // every primitive attribute on this class will be represented as column in animal table
+  // every primitive attribute on this class will be represented as column in Product table
   private String basicInfo;
```

Comment:
I think we do not need the `basicInfo` attribute anymore, because we have the `descrption` now. So you can remove it from here and the corresponding transfer objects